index.html

```diff
@@ -97,6 +98,7 @@ <h2>Release Calendar</h2>
 				</td>
 				<td>
 					<ul>
+						<li><a href="/releases/goerli/1.9.9/index.html">Release 1.9.9</a> (Wednesday, 2021-10-27)</li>
 						<li><a href="/releases/goerli/1.9.7/index.html">Release 1.9.7</a> (Friday, 2021-10-08)</li>
```

Comment:
Did we miss 1.9.8 ?

Reply:
There was never a 1.9.8 release on Goerli AFAIU, and we skipped from 1.9.7 to 1.9.9 to keep consistency with mainnet.